docs/en-us/release/faq.md

```diff
@@ -618,4 +617,92 @@ A：By deploying different worker in different yarn clusters，the steps are as
 
 ---
 
-## We will collect more FAQ later
+## Q：Update process definition error: Duplicate key TaskDefinition
+
+A：Before DS 2.0.4, there may be a problem of duplicate keys TaskDefinition due to version switching, which may cause the update workflow to fail; you can refer to the following SQL to delete duplicate data, taking MySQL as an example: (Note: Before operating, be sure to back up the original data, the SQL from pr[#8408](https://github.com/apache/dolphinscheduler/pull/8408))
```

Comment:
should we also notify users they should only run this SQL unless they use version after 2.0.0-alpha and before 2.0.4? I they upgrade from 1.3.X, they should not be wrong about that.

Reply:
I had added it.